app/models/job.rb

```diff
@@ -26,6 +26,18 @@ class Job < ApplicationRecord
   validates :status, :expire_at, presence: true
 
   scope :newest, ->{order(created_at: :asc)}
+  scope :search_by_salary, (lambda do |salary|
+    where("salary BETWEEN ? AND ?", salary.min_salary, salary.max_salary)
+  end)
+  scope :search_by_company_ids, (lambda do |company_ids|
+    where("company_id IN (?)", company_ids)
```

Comment:
```suggestion
    where("company_id IN (?)", company_ids) if company_ids.present?
```
Thử kiểu này đi em để ở hàm trên kia đỡ phải if bên ngoài